src/components/InterfacesList.js

```diff
@@ -58,10 +58,13 @@ const InterfacesList = ({ interfaces = [], connections = [] }) => {
         return iface.addresses.map(i => i.local).join(', ');
     };
 
-    const renderAlertIcon = (iface) => {
-        if (!iface.error) return;
-
-        return <><AlertIcon /></>;
+    const renderStatusIcon = (iface, conn) => {
+        console.log("IFACE; pending", iface.pending, "error", iface.error);
+        if (iface.error) {
+            return <><AlertIcon /></>;
+        } else if (iface.pending) {
+            return <><Spinner size="md" /></>;
+        }
```

Comment:
I fear an if/else if is not valid here because we'd like to stop displaying the spinner as soon as the interface _is not pending_.

```suggestion
        if (iface.error) return <><AlertIcon /></>;
        if (iface.pending) return <><Spinner size="md" /></>;
```

Reply:
I would say that both are equivalent. Or am I missing something?

Reply:
Nope, you're right. The problem was actually in a missing `pending: false`. And, believe or not, I was reading `if/else` here all the time even when I wrote `if/else if` above :man_facepalming:

Reply:
No problem :+1: